### ISSUE TITLE
[Feature request] to Create a Log File Like system's default Notepad

### ISSUE THREAD

**Issue by kevinfszu** [JasonStein/Notepads#197]
**Describe the solution you'd like**
When you next open the file, note that the date and time have been appended to the end of the log, immediately preceding the place where new text can be added. You can use this functionality to automatically add the current date and time to each log entry.

**Describe alternatives you've considered**
[How to Use Notepad to Create a Log File](https://support.microsoft.com/es-ve/help/260563/how-to-use-notepad-to-create-a-log-file)

**Comment by JasonStein** [comment]
I do not think this is a feature that will benefit the majority of the user base at this moment. I can keep this thread open to hear more from the community.

**Comment by soumyamahunt** [comment]
> **Describe the solution you'd like**
> When you next open the file, note that the date and time have been appended to the end of the log, immediately preceding the place where new text can be added. You can use this functionality to automatically add the current date and time to each log entry.
> 
> **Describe alternatives you've considered**
> [How to Use Notepad to Create a Log File](https://support.microsoft.com/es-ve/help/260563/how-to-use-notepad-to-create-a-log-file)

I have implemented it in this [PR](https://github.com/JasonStein/Notepads/pull/322).
